cpp/src/wholememory_ops/functions/nvshmem_gather_scatter_func.cuh

```diff
@@ -315,9 +315,8 @@ void nvshmem_gather_temp_get_mem_sort_idx_func(wholememory_comm_t wm_comm,
                                                wholememory_matrix_description_t output_desc,
                                                size_t embedding_entry_count_per_rank,
                                                wholememory_env_func_t* p_env_fns,
-                                               cudaStream_t stream
-
-)
+                                               cudaStream_t stream,
+                                               int gather_sms)
```

Comment:
add line `void (gather_sms);` to avoid  warning about unused variables.

Reply:
fixed